db/migrate/20170522104534_regroup_course_settings.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class RegroupCourseSettings < ActiveRecord::Migration
```

Comment:
Random question, does this migration take a long time? Do we need to do any optimisations for this?